src/App.tsx

```diff
@@ -17,18 +62,30 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onFilterBy={setFilterBy}
+                filterBy={filterBy}
+                onQuery={setQuery}
+                query={query}
```

Comment:
The event handler naming convention is not followed here. According to the [convention](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364), the event handlers should be named as `onEventName`, where `EventName` is the name of the event. In this case, `onFilterBy` and `onQuery` should be renamed to something like `onFilterByChange` and `onQueryChange` respectively.